Signum.React/JsonConverters/MListJsonConverter.cs

```diff
@@ -75,64 +75,68 @@ static MList<T> ReadJsonInternal<T>(JsonReader reader, IMListPrivate<T> existing
             var rowIdType = GetRowIdTypeFromAttribute(pr);
 
 
-            reader.Assert(JsonToken.StartArray);
+            if (reader.TokenType == JsonToken.StartArray)

```

Comment:
Why this change? looks like the only difference is the Assert -> If, everything else is formatting but still... Why an MList could be serialized as something different than `[...]`.